ONSdigital/ras-party
# making environment variables consistent

# Motivation and Context
This change sets all the environment variables consistent across all the services to alleviate confusion when adding new functionality. This will also remove confusion for migration.

# What has changed
All environment variables that have the prefix RM or RAS the prefix has been removed. 
The variable named RAS_PUBLIC_WEBSITE has now been renamed FRONTSTAGE to match all other services. The variable named COLLEX has also been renamed to COLLECTION_EXERCISE as per team agreement.

# How to test?
Run the tests as normal using make test to ensure no tests have been broken through the renaming process.

# Links
https://trello.com/c/3FBH3nNx
